lfs/config.go

```diff
@@ -40,24 +42,22 @@ var (
 
 func NewConfig() *Configuration {
 	c := &Configuration{
-		CurrentRemote:  defaultRemote,
-		isTracingHttp:  len(os.Getenv("GIT_CURL_VERBOSE")) > 0,
-		isLoggingStats: len(os.Getenv("GIT_LOG_STATS")) > 0,
+		CurrentRemote: defaultRemote,
+		envVars:       make(map[string]string),
 	}
+	c.isTracingHttp = len(c.Getenv("GIT_CURL_VERBOSE")) > 0
+	c.isLoggingStats = len(c.Getenv("GIT_LOG_STATS")) > 0
```

Comment:
Since this is already caching the bool check, we can use `os.Getenv()` here, and not worry about caching it in the `gitConfig` map. Perhaps a micro optimization that doesn't really matter.

Also, should bool checks actually look at the value, instead of seeing if the string is non empty? Common accepted values could be `1`, `true`, or `on`, while values like `0`, `false`, and `off` could still be false.